cdap-ui/app/cdap/components/AbstractWidget/RadioGroupWidget/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { objectQuery } from 'services/helpers';
+
+import withStyles, { WithStyles } from '@material-ui/core/styles/withStyles';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+
+import ThemeWrapper from 'components/ThemeWrapper';
+import If from 'components/If';
+
+export const styles = () => {
+  return {
+    inlineRadio: {
+      display: 'flex',
+      'flex-direction': 'row',
+    },
+    labelControl: {
+      marginBottom: 0,
+    },
+  };
+};
+interface IWidgetAttribute {
+  layout: string;
+  options: IOptions;
+  default: string;
+}
+interface IConfig extends Object {
+  'widget-attributes': IWidgetAttribute[];
+  name: string;
+}
+interface IOption {
+  id: string;
+  label: string;
+}
+interface IOptions extends Array<IOption> {}
+interface IRadioGroupWidgetProps extends WithStyles<typeof styles> {
+  value: string;
+  config: IConfig;
```

Comment:
instead of passing all of the configs, call it out specifically what you need in the props